xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/macros.vm

```diff
@@ -387,18 +387,19 @@ $xwiki.getUserName("xwiki:${username}")
   #end
   #if(!$picturelist.containsKey("${username}--${size}"))
     #getUserAvatarURL($username $avatarURL $size)
+    #set ($userName = $xwiki.getUserName($username, false))
```

Comment:
Looks this is just codestyle (and a little bit performances) since it does not seems to have any impact on the produced html.

Reply:
Indeed, it used to be useful but the logic has been moved elsewhere, reverting.